.github/workflows/meeting-reminder.yaml

```diff
@@ -0,0 +1,12 @@
+name: Send out reminder about monthly meeting
+
+on:
+  schedule:
+    cron: 1 0 11-17 * 5
+
+uses: cstuder/apprise-ga@master
+with:
+  title: "📡 Nächsten Montag ist wieder Freifunk Stammtisch 📡"
+  message: "Wir treffen uns ab 19:30 in den Räumen des Netz39 e.V."
```

Comment:
Wie wäre es mit:
"Das nächste Freifunk-Treffen findet am kommenden Montag um 19:30 Uhr in den Räumen des Netz39 e.V. statt."?

Es gibt unterschiedliche Interpretationen für "nächster" und manche MUAs stellen den Betreff sehr weit weg vom Body dar, sodass es ganz gut ist, die Nachricht self-contained zu machen.